config/locales/en.yml

```diff
@@ -669,6 +669,12 @@ en:
       view_more_samples: View more samples
     puid:
       title: 'Copy to clipboard value: %{puid}'
+    samples:
+      table_component:
+        counts:
+          status:
+            one: "%{selected} of %{total} sample selected"
```

Comment:
Since this is for only 1 sample selected, we can probably just set this message to `1 of 1 sample selected`